contracts/amm/AmmDataProvider.sol

```diff
@@ -167,7 +169,7 @@ contract AmmDataProvider is IAmmDataProvider {
         uint256 bTokenAmount,
         uint256 collateralTokenBalance,
         uint256 bTokenPrice
-    ) external view override returns (uint256) {
+    ) public view override returns (uint256) {
```

Comment:
We changed a few of these from external to public was there a reason for this? External I believe is more optimized for gas usage according to this article [https://gus-tavo-guim.medium.com/public-vs-external-functions-in-solidity-b46bcf0ba3ac](https://gus-tavo-guim.medium.com/public-vs-external-functions-in-solidity-b46bcf0ba3ac).